deeppavlov/models/embedders/transformers_embedder.py

```diff
@@ -52,7 +52,9 @@ def load(self):
         self.model = transformers.BertModel.from_pretrained(self.load_path, config=self.config).eval().to(self.device)
```

Comment:
`return_dict=False` should be added here, I had errors in `sentiment_twitter_bert_emb.json` and `ner_ontonotes_bert_emb.json` during `deeppavlov train ...` because code returns instance of ` BaseModelOutputWithPoolingAndCrossAttentions` instead of tuple
or I can rewrite my code as yours

Reply:
See my comments regarding this issue in your PR. My opinion is that it's preferable to use these dict-like `ModelOutput` structures because they're a bit more intuitive to work with (getting loss with `.loss` instead of `[0]` is much more concise and readable). Moreover, I have other changes in this PR that are dependent on these structures.